### PR TITLE
[9.0][MIG] Reports ported: VAT Listing and VAT Intra

### PR DESCRIPTION
**Pull request by robinkeunen**
Migration from `10.0/l10n_be_vat_reports` to 9.0.

Based on #57 and #74

### CODE REVIEW

File: l10n_be_vat_reports/wizard/l10n_be_vat_intra_view.xml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                        <button name="create_xml" string="Create _XML" type="object" class="oe_highlight"/>
+                        <button name="preview" string="_Preview" type="object" class="oe_highlight" />
+                        or
+                        <button string="Cancel" class="oe_link" special="cancel" />
+                    </footer>
+                </form>
+            </field>
+        </record>
+
+        <record id="view_vat_intra_save" model="ir.ui.view">
```
Comment (tbaden) [inline l10n_be_vat_reports/wizard/l10n_be_vat_intra_view.xml:39]: optional:
id pattern

> For a view: <model_name>_view_<view_type>, where view_type is kanban, form, tree, search, ...

File: l10n_be_vat_reports/wizard/l10n_be_vat_intra_view.xml [hunk 2]
```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+        <record id="view_vat_intra" model="ir.ui.view">
```
Comment (tbaden) [inline l10n_be_vat_reports/wizard/l10n_be_vat_intra_view.xml:5]: optional:
id pattern

> For a view: <model_name>_view_<view_type>, where view_type is kanban, form, tree, search, ...

File: l10n_be_vat_reports/wizard/l10n_be_partner_vat_listing.xml [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+            <field name="name"/>
+            <field name="vat"/>
+            <field name="turnover"/>
+            <field name="vat_amount"/>
+          </group>
+        </form>
+      </field>
+    </record>
+
+    <record id="view_vat_listing_tree" model="ir.ui.view">
```
Comment (tbaden) [inline l10n_be_vat_reports/wizard/l10n_be_partner_vat_listing.xml:99]: optional:
id pattern

> For a view: <model_name>_view_<view_type>, where view_type is kanban, form, tree, search, ...

File: l10n_be_vat_reports/wizard/l10n_be_partner_vat_listing.xml [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+            <separator string="XML File has been Created." colspan="4"/>
+            <label string="Save the File with '.xml' extension." colspan="4" nolabel="1"/>
+            <field name="name"/><newline/>
+            <field name="file_save"/>
+        </form>
+      </field>
+    </record>
+
+    <!-- osv_memory for vat listing of clients -->
+    <record id="view_vat_listing_form" model="ir.ui.view">
```
Comment (tbaden) [inline l10n_be_vat_reports/wizard/l10n_be_partner_vat_listing.xml:84]: optional:
id pattern

> For a view: <model_name>_view_<view_type>, where view_type is kanban, form, tree, search, ...

File: l10n_be_vat_reports/wizard/l10n_be_partner_vat_listing.xml [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+              <group colspan="4">
+                <button colspan="2" name="create_xml" string="Create XML" type="object" icon="fa-cogs"/>
+                <button colspan="2" name="print_vatlist" string="Print" type="object"  icon="fa-print"/>
+              </group>
+          </group>
+        </form>
+      </field>
+    </record>
+
+    <record id="view_vat_listing_result" model="ir.ui.view">
```
Comment (tbaden) [inline l10n_be_vat_reports/wizard/l10n_be_partner_vat_listing.xml:64]: optional:
id pattern

> For a view: <model_name>_view_<view_type>, where view_type is kanban, form, tree, search, ...

File: l10n_be_vat_reports/wizard/l10n_be_partner_vat_listing.xml [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+      <field name="res_model">partner.vat</field>
+      <field name="view_type">form</field>
+      <field name="view_mode">form</field>
+      <field name="view_id" ref="view_partner_vat_listing"/>
+      <field name="target">new</field>
+    </record>
+
+    <menuitem name="Annual Listing Of VAT-Subjected Customers" parent="l10n_be.account_reports_be_statements_menu" action="action_partner_vat_listing" id="partner_vat_listing"/>
+
+    <record id="view_vat_listing" model="ir.ui.view">
```
Comment (tbaden) [inline l10n_be_vat_reports/wizard/l10n_be_partner_vat_listing.xml:38]: optional:
id pattern

> For a view: <model_name>_view_<view_type>, where view_type is kanban, form, tree, search, ...

File: l10n_be_vat_reports/wizard/l10n_be_partner_vat_listing.xml [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+          <footer>
+            <button name="get_partner" string="View Customers" type="object"  class="oe_highlight"  />
+            or
+            <button string="Cancel" class="oe_link" special="cancel" />
+          </footer>
+        </form>
+      </field>
+    </record>
+
+    <record id="action_partner_vat_listing" model="ir.actions.act_window">
```
Comment (tbaden) [inline l10n_be_vat_reports/wizard/l10n_be_partner_vat_listing.xml:26]: optional:
id pattern

> For an action: the main action respects <model_name>_action. Others are suffixed with _<detail>, where detail is an underscore lowercase string explaining the action (should not be long). This is used only if multiple actions are declared for the model.

File: l10n_be_vat_reports/wizard/l10n_be_partner_vat_listing.xml [hunk 8]
```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+  <data>
+
+    <record id="view_partner_vat_listing" model="ir.ui.view">
```
Comment (tbaden) [inline l10n_be_vat_reports/wizard/l10n_be_partner_vat_listing.xml:5]: optional:
id pattern

> For a view: <model_name>_view_<view_type>, where view_type is kanban, form, tree, search, ...

File: l10n_be_vat_reports/__openerp__.py [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+#     You should have received a copy of the GNU Affero General Public License
+#     along with l10n_be_mis_reports.
+#     If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': 'Belgium VAT Reports',
+    'author': 'ACSONE SA/NV,'
+              'Odoo Community Association (OCA)',
+    'website': 'http://acsone.eu',
```
Comment (tbaden) [inline l10n_be_vat_reports/__openerp__.py]: please set website to: https://github.com/OCA/l10n-belgium

**Review by tbaden — CHANGES_REQUESTED** [review]
thanks for the work 👍 
I did a code review: just some small changes:
please use the new fragment [readme](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-9.0)

**Comment by robinkeunen** [comment]
Thanks for the review, @tbaden. I used readme fragments and fixed the website. However, I did not change the xml id as I don't want to risk breaking code using these ids. But thanks for the recommendations, I always wonder how to name the views consistently.

**Comment by robinkeunen** [comment]
I also rebased on `oca/9.0`

**Comment by remytms** [comment]
@tbaden Is it possible to merge this PR ? :)

**Review by houssine78 — APPROVED** [review]

**Comment by houssine78** [comment]
@sbidoul I think this PR is ready for merging :) I ping you as it stayed without response after taken the comment into account ci and runbot are green and we use it in production since then.

Thanks and happy new year !

**Comment by sbidoul** [comment]
@houssine78 we still miss the correct attribution to Odoo unless I miss somehintg. I commented in #75: let's finish that one first.

Happy new year!

**Review by remytms — APPROVED** [review]
LGTM

**Comment by houssine78** [comment]
@sbidoul I fixed to correct attribution to odoo and provided a fix. Could you tell me if we're good ?

**Pull request closed without merging**